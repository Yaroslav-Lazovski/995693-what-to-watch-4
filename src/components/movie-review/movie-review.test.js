import React from "react";
import renderer from "react-test-renderer";

import MovieReview from "./movie-review.jsx";


const review = {
  id: 1,
  user: {
    id: 4,
    name: `User`
  },
  comment: `Comment`,
  date: `December 24, 2016`,
  rating: 7.1,
};


describe(`MovieReview Snapshot`, () => {
  it(`Should MovieReview render correctly`, () => {
    const tree = renderer
      .create(<MovieReview
        review={review}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
