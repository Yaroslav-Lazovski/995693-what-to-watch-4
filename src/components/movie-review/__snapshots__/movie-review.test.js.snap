// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieReview Snapshot Should MovieReview render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Comment
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        User
      </cite>
      <time
        className="review__date"
        dateTime="December 24, 2016"
      >
        December 24, 2016
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    7.1
  </div>
</div>
`;
